Adamant/Modules/Chat/View/ChatViewController.swift

```diff
@@ -144,6 +145,9 @@ final class ChatViewController: MessagesViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         defer { viewAppeared = true }
+        
+        unreadChatsCounter.updateCounter(count: viewModel.unreadChatsCount)
```

Comment:
Overhead. You already subscribe for updates in the `setupObservers()` function

`viewModel.$unreadChatsCount...`

Reply:
move to viewModel